### PR TITLE
Edit Group Chat

### PR DESCRIPTION
**Pull request by imphungky**
## DEV-004

Fixes: #

- [X] Has the code been formatted?
- [ ] Have dependencies/packages been added/removed?
      If so, please list them.

### Changes

List out all the changes, and then explain the changes made at a higher level

- Modified chat/[id].jsx to contain the id and check user permissions for editting
- created a new form for editing containing current chat info
- modified query to fetch for more group chat information for editing

### CODE REVIEW

File: client/content/locale/en.json
Comment (jcserv) [inline client/content/locale/en.json:47]: Could you also add these to fr.json?

File: client/pages/chat/[id].jsx
Comment (jcserv) [inline client/pages/chat/[id].jsx]: Nice! Do you think we could abstract this into something like a permission wrapper component?
I have no clue tbh I'm just wondering if we could apply this for other features

File: server/resolvers/GroupChatResolver.ts
Comment (jcserv) [inline server/resolvers/GroupChatResolver.ts]: unused

File: server/resolvers/GroupChatResolver.ts
Comment (jcserv) [inline server/resolvers/GroupChatResolver.ts]: I think you can change returns to () to remove the eslint warning

File: client/components/DeleteChatModal.jsx
Comment (jcserv) [inline client/components/DeleteChatModal.jsx:69]: can we get a toast for when the delete is unsuccessful as well?

File: client/components/DeleteChatModal.jsx
Comment (jcserv) [inline client/components/DeleteChatModal.jsx:87]: currently this content isn't populating for me

**Review by jcserv — CHANGES_REQUESTED** [review]
Great work, just a bit of feedback but after that I think it's good to go - pending a review from @AipioxTechson

**Review by AipioxTechson — COMMENTED** [review]
Aside from Jerry's requested changes, it looks good!

File: client/components/DeleteChatModal.jsx
Comment (jcserv) [inline client/components/DeleteChatModal.jsx:57]: Can you redirect the user to the homepage after deleting?

**Pull request merged**
